modules/ROOT/pages/subqueries/subqueries-in-transactions.adoc

```diff
@@ -550,3 +550,83 @@ RETURN largeLists
 1+d|Rows: 2
 
 |===
+
+[[composite-databases]]
+== Composite databases
+`CALL { ... } IN TRANSACTIONS` works on composite databases.
+Composite databases are virtual databases used for querying their constituents, so the examples above are generally
+valid for composite databases, but they must be slightly modified to accommodate the nature of composite databases.
```

Comment:
```suggestion
Composite databases are virtual databases used for querying their constituents.
The examples above are generally valid for composite databases, but they must be slightly modified to accommodate the nature of composite databases.
```

Reply:
This suggestion was committed. I don't know why it is not shown as so here.